### PR TITLE
Show error panel if user invokes clipper on local pdf in non-chrome, …

### PR DESCRIPTION
**Pull request by mttwc**
…disable bookmark mode on local files

### CODE REVIEW

**Comment by VishaalK** [comment]
Testing meow

**Review by VishaalK — APPROVED** [review]
📦

File: src/scripts/clipperUI/components/modeButtonSelector.tsx [hunk 1]
```diff
@@ -100,6 +100,9 @@ class ModeButtonSelectorClass extends ComponentBase<{}, ClipperStateProp> {
 	}
 
 	private getBookmarkModeButton(currentMode: ClipMode) {
+		if (this.props.clipperState.pageInfo.rawUrl.indexOf("file:///") === 0) {
```
Comment (iconix) [inline src/scripts/clipperUI/components/modeButtonSelector.tsx:103]: what about that helper function we were talking about? :)

**Review by leakymemory — APPROVED** [review]

**Review by iconix — APPROVED** [review]
👍

**Pull request merged**